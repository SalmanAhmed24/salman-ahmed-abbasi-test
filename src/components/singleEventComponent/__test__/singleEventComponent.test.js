import React from 'react';
//import {  BrowserRouter } from 'react-router-dom';
import SingleEventComponent from '../singleEventComponent';
import "@testing-library/jest-dom/extend-expect";
import renderer from 'react-test-renderer';


test('<SingleEventComponent> testing',()=>{
    const venue = 
        {
            country:"pakistan",
            city:"islamabd",
            name:"name",
        }
    

    const component = renderer.create(<SingleEventComponent venue={venue} date="mm-dd-yyyy"/>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();    
})