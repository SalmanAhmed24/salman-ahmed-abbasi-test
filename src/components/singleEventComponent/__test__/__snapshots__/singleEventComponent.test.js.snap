// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SingleEventComponent> testing 1`] = `
<div
  className="event-card"
>
  <p
    className="event-h"
  >
    Event Details
  </p>
  <div
    className="sub-info-wrap"
  >
    <div
      className="sub-info"
    >
      <p
        className="sub-h"
      >
        Country
      </p>
      <p
        className="sub-p"
      >
        pakistan
      </p>
    </div>
    <div
      className="sub-info"
    >
      <p
        className="sub-h"
      >
        City
      </p>
      <p
        className="sub-p"
      >
        islamabd
      </p>
    </div>
  </div>
  <div
    className="sub-info-wrap"
  >
    <div
      className="sub-info"
    >
      <p
        className="sub-h"
      >
        Venue
      </p>
      <p
        className="sub-p"
      >
        name
      </p>
    </div>
    <div
      className="sub-info"
    >
      <p
        className="sub-h"
      >
        Date
      </p>
      <p
        className="sub-p"
      >
        mm-dd-yyyy
      </p>
    </div>
  </div>
</div>
`;
